src/display.rs

```diff
@@ -966,5 +970,12 @@ mod tests {
             writer.write_all(&vec[..]).unwrap();
             assert_eq!(writer.into_inner(), vec.to_lower_hex_string());
         }
+
+        #[test]
+        fn hex_writer_accepts_and_mut() {
+            let mut dest_buf = ArrayString::<64>::new();
+            let mut dest = HexWriter::new(&mut dest_buf, Case::Lower);
+            let _got = dest.write(b"some data").unwrap();
```

Comment:
This test is actually testing that `std` still has the `&mut T` impl. I'm certain we don't need to test that. :)

Reply:
Correct. It also proves, probably unnecessarily like you say, that the docs I wrote in this PR are correct.